app/controllers/application_controller.rb

```diff
@@ -24,6 +24,14 @@ def admin_signed_in?
     session.key?(:user_id)
   end
 
+  def admin_session
+    AdminSession.new(session[:user_id], session[:organisation_id], session[:role_codes])
+  end
+
+  def service_operator_signed_in?
```

Comment:
I think we should define this in the base admin controller to keep it scoped to that part of the codebase.